addons/web/static/src/views/fields/statusbar/statusbar_field.scss

```diff
@@ -52,7 +52,7 @@
 
             &:disabled {
                 opacity: 1;
-                pointer-events: auto;
+                pointer-events: none;
```

Comment:
Why not "just" remove this rule as Bootstrap already set it to `none` ?

Reply:
Indeed 👍 
Done, thanks !